gearcmd/worker.go

```diff
@@ -114,6 +114,9 @@ func (conf TaskConfig) doProcess(job baseworker.Job) error {
 		// Will be nil if the channel was closed without any errors
 		return err
 	case <-time.After(conf.CmdTimeout):
+		if err := cmd.Process.Kill(); err != nil {
```

Comment:
Should we also add signal handling? I'm having trouble finding what happens to unhandled signals in Go. 

here is an example of how some sigals can be handled: https://github.com/mindreframer/golang-devops-stuff/blob/master/src/github.com/bradfitz/runsit/runsit.go#L654-L672